static/js/context-menu.js

```diff
@@ -245,6 +245,51 @@ class MenuItem {
   }
 }
 
+class SearchMenuItem extends MenuItem {
+  constructor(options) {
+    super({
+      html: ContextMenu.fmt("Search for <strong>_</strong>", options.label),
+      href: "",
+      section: "symbol-searches",
+      icon: "search",
+      confidence: options.confidence,
+    });
+
+    this.label = options.label;
+    this.tree = options.tree;
+    this.syms = options.syms;
+    this.def = options.def;
+
+    this.updateHref();
+  }
+
+  updateHref() {
+    const syms = encodeURIComponent(this.syms.join(","));
+    this.href = `/${this.tree}/search?q=symbol:${syms}&redirect=false`;
+  }
+
+  toKey() {
+    if (this.def) {
+      // This is mergeable.
+      // Do not put syms in the key, and merge them later.
+      return JSON.stringify({
+        label: this.label,
+        icon: this.icon,
+        def: this.def,
+        confidence: this.confidence,
+      });
+    }
+    // Not mergeable.
+    return super.toKey();
+  }
+
+  merge(other) {
+    console.log(this.syms, other.syms);
```

Comment:
Did you mean to leave this in?  If so, it's probably nice to add a human-readable label string to the start.